HW-11/homework11.py

```diff
@@ -0,0 +1,56 @@
+import json
+data = {
+    'Frodo': '12345678',
+    'Sam': '87654321',
+    'Mari': '18273645',
+
+}
+
+
+try:
+    with open(r'C:\HW\HomeWork\HW-11\file', 'x') as file:
+        data = json.dumps(data)
+        file.write(data)
+        print('Open file')
+except FileExistsError:
+    print('Open file')
+
+while True:
+    with open(r'C:\HW\HomeWork\HW-11\file', 'r+') as file:
```

Comment:
You don't need to read data from the file every iteration. It is enough to do it once before your main loop is started